admin/used_flusher.go

```diff
@@ -73,24 +100,48 @@ func (u *usedFlusher) flush() {
 	u.mu.Lock()
 	deployments := u.deployments
 	u.deployments = make(map[string]bool)
+
+	users := u.users
+	u.users = make(map[string]bool)
+
+	userTokens := u.userTokens
+	u.userTokens = make(map[string]bool)
+
+	serviceTokens := u.serviceTokens
+	u.serviceTokens = make(map[string]bool)
 	u.mu.Unlock()
```

Comment:
Slight optimization suggestion: only call `make` if the previous one was not empty. I.e. wrap the `u.xxx = ...` calls with `if len(u.xxx) > 0 { ...`